.github/workflows/docker-build-test.yaml

```diff
@@ -300,7 +300,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: aptos-node-v1.9.5 # test against a previous testnet release
+      IMAGE_TAG: adhoc-20240328 # test against a previous testnet release
```

Comment:
You should probably test against the v1.10 since adhoc and that are compatible, but either way